lib/diplomat/query.rb

```diff
@@ -32,7 +32,9 @@ def create(definition, options = {})
       custom_params = options[:dc] ? use_named_parameter('dc', options[:dc]) : nil
       @raw = send_post_request(@conn, ['/v1/query'], options, definition, custom_params)
       parse_body
-    rescue Faraday::ClientError
+    rescue Faraday::TimeoutError => e
```

Comment:
I would not put this, simply let the exception Faraday::Timeout error raise by itself... my example with timeout was a bad example, I was talking on ClientError on Faraday 1+

So, remove the :
```
    rescue Faraday::TimeoutError => e
        raise e
```